bigquery-writer/bigquery-writer.py

```diff
@@ -97,38 +160,37 @@ def start(self):
             sleep(1000)
             return
```

Comment:
If there is no sleep, the container will just keep restarting, which just sucks resources.